### DISCUSSION TITLE
field-service-16.0

### DISCUSSION THREAD

**Discussion by MarcosGraficon** [OCA/odoo-community.org#136] [Q&A] [answered]
Hellow guys, I need a bit of help, I'm working in odoo 15 developing some extensions, i I have never installed an external extension, and need this one, but it seems that is in Odoo 16, isn't it? So i want to adapt it to 15, the only thing maybe is to swap all the 16.0 to 15.0 and then how do i install them? Thank u so much

**Comment by chienandalu** [comment, marked as answer]
You have to go to the proper branch and there you'll have the modules available for your version: https://github.com/OCA/field-service/tree/15.0

Installing modules from another version normally doesn't go well, as the smallest incompatibility could break it.

**Reply by MarcosGraficon** [reply]
Ohhh didnt see that, Ty
